examples/grpc-service-basic-auth-file-user-store/grpc_service_basic_auth_file_user_store.md

```diff
@@ -32,12 +32,23 @@ Once you run the command, the `grpc_service_pb.bal` file is generated inside the
 
 3. Create a new `grpc_service_basic_auth_file_user_store.bal` Ballerina file inside the `service` package and add the service implementation.
 
+    >**Tip:** You may need to change the certificate file path and private key file path in the code below.  
+
    ::: code grpc_service_basic_auth_file_user_store.bal :::
 
-4. As a prerequisite, ensure that the `Config.toml` file is populated correctly with the user information.
+4. As a prerequisite, execute the command below to populate the `Config.toml` file correctly with the user information.
 
-6. Execute the commands below to build and run the `service` package.
+    ```bash
+    $ echo '[[ballerina.auth.users]]
+    username="alice"
+    password="password1"
+    scopes=["scope1"]
+    [[ballerina.auth.users]]
+    username="bob"
+    password="password2"
+    scopes=["scope2", "scope3"]' > Config.toml
```

Comment:
Shall we just add the configs which need to add in `Config.toml`, rather than executing as bash script

Reply:
isn't it easy for the users to run the bash script rather than finding the `Config.toml` file and adding the configs into it?

Reply:
Done.